djstripe/stripe_objects.py

```diff
@@ -375,29 +376,18 @@ def status(self):
             return "Closed"
         return "Open"
 
-    @classmethod
-    def stripe_object_to_record(cls, data):
-        period_end = convert_tstamp(data, "period_end")
-        period_start = convert_tstamp(data, "period_start")
-        date = convert_tstamp(data, "date")
-
-        return {
-            "stripe_id": data["id"],
-            "attempted": data["attempted"],
-            "closed": data["closed"],
-            "paid": data["paid"],
-            "period_end": period_end,
-            "period_start": period_start,
-            "subtotal": data["subtotal"] / decimal.Decimal("100"),
-            "total": data["total"] / decimal.Decimal("100"),
-            "date": date,
-            "charge": data.get("charge") or "",
-        }
-
     def sync(self, data=None):
         for attr, value in data.items():
             setattr(self, attr, value)
 
+    @classmethod
+    def stripe_object_to_record(cls, data):
+        # Perhaps meaningless legacy code. Nonetheless, preserve it. If charge is null
+        # then convert it to ""
+        if 'charge' in data and data['charge'] is None:
+            data['charge'] = ""
```

Comment:
Yeah, it's legacy API code. Setting it to None is fine after API version 2013-08-12